### ISSUE TITLE
TypeError in webpack-plugin 5.2.0

### ISSUE THREAD

**Issue by willdurand** [smooth-code/loadable-components#168]
## 💥 Regression Report

Latest webpack plugin version (`5.2.0`) throws an error when running with the dev server (middleware) and `writeToDisk` is set to `true`. There is no error when setting this option to `false`.

```
[0] /Users/williamdurand/projects/mozilla/addons-frontend/node_modules/@loadable/webpack-plugin/lib/index.js:64
[0]       if (_this.isRequestFromDevServer() && _this.compiler.options.devServer) {
[0]                                                            ^
[0]
[0] TypeError: Cannot read property 'options' of undefined
[0]     at LoadablePlugin.getManifestOutputPath (/Users/williamdurand/projects/mozilla/addons-frontend/node_modules/@loadable/webpack-plugin/lib/index.js:64:60)
[0]     at LoadablePlugin.writeAssetsFile (/Users/williamdurand/projects/mozilla/addons-frontend/node_modules/@loadable/webpack-plugin/lib/index.js:80:28)
[0]     at /Users/williamdurand/projects/mozilla/addons-frontend/node_modules/@loadable/webpack-plugin/lib/index.js:43:15
[0]     at AsyncSeriesHook.eval [as callAsync] (eval at create (/Users/williamdurand/projects/mozilla/addons-frontend/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:7:1)
[0]     at AsyncSeriesHook.lazyCompileHook (/Users/williamdurand/projects/mozilla/addons-frontend/node_modules/tapable/lib/Hook.js:154:20)
[0]     at Compiler.emitAssets (/Users/williamdurand/projects/mozilla/addons-frontend/node_modules/webpack/lib/Compiler.js:363:19)
[0]     at onCompiled (/Users/williamdurand/projects/mozilla/addons-frontend/node_modules/webpack/lib/Watching.js:50:19)
[0]     at hooks.afterCompile.callAsync.err (/Users/williamdurand/projects/mozilla/addons-frontend/node_modules/webpack/lib/Compiler.js:552:14)
[0]     at AsyncSeriesHook.eval [as callAsync] (eval at create (/Users/williamdurand/projects/mozilla/addons-frontend/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:15:1)
[0]     at AsyncSeriesHook.lazyCompileHook (/Users/williamdurand/projects/mozilla/addons-frontend/node_modules/tapable/lib/Hook.js:154:20)
[0]     at compilation.seal.err (/Users/williamdurand/projects/mozilla/addons-frontend/node_modules/webpack/lib/Compiler.js:549:30)
[0]     at AsyncSeriesHook.eval [as callAsync] (eval at create (/Users/williamdurand/projects/mozilla/addons-frontend/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:6:1)
[0]     at AsyncSeriesHook.lazyCompileHook (/Users/williamdurand/projects/mozilla/addons-frontend/node_modules/tapable/lib/Hook.js:154:20)
[0]     at hooks.optimizeAssets.callAsync.err (/Users/williamdurand/projects/mozilla/addons-frontend/node_modules/webpack/lib/Compilation.js:1323:35)
[0]     at AsyncSeriesHook.eval [as callAsync] (eval at create (/Users/williamdurand/projects/mozilla/addons-frontend/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:6:1)
[0]     at AsyncSeriesHook.lazyCompileHook (/Users/williamdurand/projects/mozilla/addons-frontend/node_modules/tapable/lib/Hook.js:154:20)
```

## Last working version

Worked up to version: `5.0.2`

Stopped working in version: `5.2.0`

## To Reproduce

- Upgrade the plugin version in a project configured with `webpack-dev-middleware`?

## Expected behavior

No error, ideally :)

## Link to repl or repo (highly encouraged)

- PR: https://github.com/mozilla/addons-frontend/pull/7017 – It is not a small project, though (sorry).

**Comment by neoziro** [comment]
Hello @willdurand, thanks! Anyone to help fixing it please?

**Comment by radum** [comment]
@willdurand Can you add here the webpackconfig you have? I have dev server (middleware)  running and works fine.

**Comment by willdurand** [comment]
@radum we use this config: https://github.com/mozilla/addons-frontend/blob/master/webpack.dev.config.babel.js, which extends https://github.com/mozilla/addons-frontend/blob/master/webpack.prod.config.babel.js. The code of our dev server is here: https://github.com/mozilla/addons-frontend/blob/master/bin/webpack-dev-server.js#L36

**Comment by neoziro** [comment]
Fixed in v5.2.1
